File: Java/프로그래머스/0/181929. 원소들의 곱과 합/원소들의 곱과 합.java
class Solution {
    public int solution(int[] num_list) {
        int answer = 0;
        int mulsum=1;
        int squaresum=0;
        for(int i=0 ;i<num_list.length;i++){
            mulsum*=num_list[i];
            squaresum+=num_list[i];
        }
        if(mulsum<squaresum*squaresum) answer=1;
        else answer=0;
        return answer;
    }
}

File: JavaScript/프로그래머스/0/181929. 원소들의 곱과 합/원소들의 곱과 합.js
function solution(num_list) {
    var answer = 0;
    var mul=1;
    var sum=0;
    for(var i of num_list) {
        mul*=i;
        sum+=i;
    }
    if(mul<sum*sum) answer=1;
    return answer;
}